### ISSUE TITLE
Build a skeleton template

### ISSUE THREAD

**Issue by Riv-Con** [DiAJVince/graceshopper#6]
This is going to consist of the following:

In the master branch I am simply going to create the folder structure and a few representative empty files for our application. This assumes a React-Redux app equivalent to one we would do for a weekend project.

I am then going to create a branch from that folder structure called "template_VR". This branch will be a working template with all the files necessary to show that the application is wired correctly. It will be a baseline template that we can work from.

Once I am finished with writing and testing my branch, I will contact my team-mates so that we can decide whether to merge it with the master branch.

...accidently closed the previous task.  Re-opened here.

**Comment by Riv-Con** [comment]
Skeleton directory structure created in Master  10/4/2017

Branch created --> template_VR  Will work on working template in the morning

**Comment by Riv-Con** [comment]
At this point our template (branch: template_VR):

- runs
- builds custom.scss into custom.css
-renders our footer at port 2020
- seeds & syncs gracehopper dbms
- seeds test model

....need to:

add test routes...
 wire in react-redux store mechanisms...

**Comment by Riv-Con** [comment]
test routes are done for test module

last item on my task sheet:

wire in react-redux store mechanisms...

**Comment by Riv-Con** [comment]
This task is completed on my end.  You both need to do the following:

- review the directory structure and file structure
- clone the fully functioning branch to your local machine & npm install
- run and check that is works and is acceptable as a template.

Note:  I created a "test" model and seeded it.  I then created a "test" reducer, test action creators, "test" routes & "test" methods to make sure that the template is fully functioning.

All of this "test" functionality can be removed when we are ready to insert valid code.
